src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.props

```diff
@@ -28,15 +28,24 @@ Copyright (c) .NET Foundation. All rights reserved.
     <EnableWebSdkImplicitPackageVersions>false</EnableWebSdkImplicitPackageVersions>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <_ImageGlobs>**/*.png;**/*.bmp;**/*.jpg;**/*.dds;**/*.tif;**/*.tga;**/*.gif</_ImageGlobs>
+  </PropertyGroup>
 
   <ItemGroup Condition=" '$(EnableDefaultItems)' == 'true' ">
     <Compile Include="**/*$(DefaultLanguageSourceExtension)" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" Condition=" '$(EnableDefaultCompileItems)' == 'true' " />
     <EmbeddedResource Include="**/*.resx" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" Condition=" '$(EnableDefaultEmbeddedResourceItems)' == 'true' " />
+    <!-- WindowsAppSdk is a NuGet delivered Windows SDK. Include only if the package is installed. -->
```

Comment:
"Windows SDK" is a different thing

Reply:
How about "app SDK"? If that's not OK, what do you think is best?

Reply:
just SDK?

Reply:
The condition is based on the `WindowsAppSdkFoundation` property, not based on the NuGet package being referenced as the comment seems to suggest.  Does the NuGet package set the property?  Or is a project referencing the NuGet package expected to set the property?

It would be good to clarify the comment.